internal/cli/cmd/cluster/create.go

```diff
@@ -1318,7 +1318,13 @@ func validateDefaultSCInConfig(dynamic dynamic.Interface) (bool, error) {
 		return false, err
 	}
 	var config map[string]interface{}
+	if cfg.Object["data"] == nil {
+		return false, nil
+	}
 	data := cfg.Object["data"].(map[string]interface{})
+	if data["config.yaml"] == nil {
```

Comment:
can you provide documentation info for this `config.yaml`, I don't understand how is this part just go added. //cc @ldming

Reply:
I provide more information about `config.yaml` in the commnet now.